helm/azure-operator/templates/configmap.yaml

```diff
@@ -27,6 +27,11 @@ data:
       installation:
         name: '{{ .Values.Installation.V1.Name }}'
       {{- if .Values.Installation.V1.Guest }}
```

Comment:
Not part of this PR, but what's this if? Naming suggest like this is adaptation for CP and TC app, but it doesn't make sense for `azure-operator`. Something else?

Reply:
Not sure. Maybe it was to check if OIDC parameters were being passed?

Reply:
:man_shrugging: